locust_benchmark/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM hopsworks/locust:hsfs-latest
+
+USER root
+
+RUN apt-get update -y
+RUN apt-get -y install gcc python3-dev librdkafka-dev git
```

Comment:
Run as single command otherwise `update` will always be cached and the installs will run on outdated apt infos if the command is changed.

Reply:
fixed, thanks